app/docker/views/services/servicesController.js

```diff
@@ -21,6 +21,7 @@ function ($q, $scope, $state, Service, ServiceService, ServiceHelper, Notificati
     });
   };
 
+
```

Comment:
Please remove the extra line break here.

Reply:
no prob..will fix it and submit again.